CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.3.2
```

Comment:
Let's remove these changes. I'm not sure we need to release a new version for these kinds of changes.

Reply:
Removed